appveyor.yml

```diff
@@ -52,7 +52,7 @@ build_script:
   - travis-tool.sh install_deps
   - travis-tool.sh r_binary_install curl
   - Rscript -e "install.packages(c('covr', 'spelling'), repos = 'http://cran.us.r-project.org', type='win.binary')"
-  - Rscript -e "install.packages('https://ci.appveyor.com/api/projects/open-systems-pharmacology-ci/ospsuite-rutils/artifacts/ospsuite.utils.zip', repos = NULL, type = 'binary')"
+  - Rscript -e "download.file('https://ci.appveyor.com/api/projects/open-systems-pharmacology-ci/ospsuite-rutils/artifacts/ospsuite.utils.zip?pr=false', destfile = 'ospsuite.utils.zip', mode='wb'); install.packages('ospsuite.utils.zip', repos = NULL, type = 'binary')"

```

Comment:
seems that none of the R installing functions (`install.packages`, `remotes::install_url`, `devtools::install_url`) can properly handle remote urls with arguments (like here .../ospsuite.utils.zip**?pr=false'**).
So downloading the file first and then installing from local. 
Works so far. If somebody finds a possibility how to let the installing functions to work with urls with arguments - can be simplified

Reply:
I found the curl options to access the file on appveyor. However, the downloaded file is named `ospsuite.utils.zip?pr=false` and the functions throws an error saying there's no zip file:
```r
> install.packages("https://ci.appveyor.com/api/projects/open-systems-pharmacology-ci/ospsuite-rutils/artifacts/ospsuite.utils.zip&?pr=false",
                   repos = NULL, 
                   type="binary", 
                   method = "curl",
                   extra="-L")
```
```

Installing package into ‘C:/Users/Felix/AppData/Local/R/win-library/4.2’
(as ‘lib’ is unspecified)
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100  3420  100  3420    0     0   4051      0 --:--:-- --:--:-- --:--:--  4052
Error in install.packages : zip file ‘C:\Users\Felix\AppData\Local\Temp\RtmpYNDsvP/downloaded_packages/ospsuite.utils.zip&?pr=false’ not found
```

So unless whe change something in `remotes::install_url`, `devtools::install_url` we'll have to keep the download and install in two steps.